08_education/programs/MEPS_metrics_city_update_jsc3.do

```diff
@@ -70,86 +57,81 @@ replace city_name_edited = subinstr(city_name_edited, " city", "", .)
 drop city_name
 rename city_name_edited city_name
 
-*hardcode fixes so names merge
-replace city_name="Ventura" if city_name=="San Buenaventura (Ventura)"
-replace city_name="Athens" if city_name=="Athens-Clarke County unified government (balance)"
-replace city_name="Augusta" if city_name=="Augusta-Richmond County consolidated government (balance)"
-replace city_name="Macon" if city_name=="Macon-Bibb County"
-replace city_name="Honolulu" if city_name=="Urban Honolulu"
-replace city_name="Boise" if city_name=="Boise City"
-replace city_name="Indianapolis" if city_name=="Indianapolis city (balance)"
-replace city_name="Lexington" if city_name=="Lexington-Fayette"
-replace city_name="Louisville" if city_name=="Louisville/Jefferson County metro government (balance)"
-replace city_name="Lees Summit" if city_name=="Lee's Summit"
-replace city_name="Ofallon" if city_name=="O'Fallon"
-replace city_name="Nashville" if city_name=="Nashville-Davidson metropolitan government (balance)"
-replace city_name="Ofallon" if city_name=="O'Fallon"
-replace city_name="Mcallen" if city_name=="McAllen"
-replace city_name="Mckinney" if city_name=="McKinney"
-replace city_name="Anchorage" if city_name=="Anchorage municipality"
-
-save "intermediate/cityfile.dta", replace
-
-** get CCD enrollment **
-*add if, else command once decide how to deal with future iterations
-if $rebuild_portal_data == 1 {
+	*hardcode fixes so names merge
+	replace city_name="Ventura" if city_name=="San Buenaventura (Ventura)"
+	replace city_name="Athens" if city_name=="Athens-Clarke County unified government (balance)"
+	replace city_name="Augusta" if city_name=="Augusta-Richmond County consolidated government (balance)"
+	replace city_name="Macon" if city_name=="Macon-Bibb County"
+	replace city_name="Boise" if city_name=="Boise City"
+	replace city_name="Lexington" if city_name=="Lexington-Fayette"
+	replace city_name="Louisville" if city_name=="Louisville/Jefferson County metro government (balance)"
+	replace city_name="Nashville" if city_name=="Nashville-Davidson metropolitan government (balance)"
+	replace city_name="Mcallen" if city_name=="McAllen"
+	replace city_name="Mckinney" if city_name=="McKinney"
+		
+	save "intermediate/cityfile.dta", replace // gitignore
+
+*****************************
+*****CCD District Data*******
+*****************************
+** Download CCD school level enrollment by race/ethnicity data from Urban's Education Data Portal **
 educationdata using "school ccd enrollment race", sub(year=2014:${year}) csv clear
 save "raw\ccd_enr_2014-${year}.dta", replace
-}
-if $rebuild_portal_data == 0 {
-	use "${raw_data}ccd_enr_2014-${year}.dta", clear
-}
+numlabel, add
 
-keep if grade==99 & sex==99
-drop leaid ncessch_num grade sex fips
+replace enrollment = . if enrollment < 0
 
-replace enrollment = 0 if enrollment < 0
+keep if grade==99 
```

Comment:
In this chunk you are keeping if `grade==99` and dropping if `sex==99`. This caught my eye because usually 99 is an internal code for missing or refused or an unknown value. In which case dropping where `sex==99` makes sense but I don't know why you would keep if `grade==99`. 

If this is correct please add a comment explaining what 99 means in this context. Thanks.